modules/cli-options/src/main/scala/scala/cli/commands/SharedDebugOptions.scala

```diff
@@ -0,0 +1,28 @@
+package scala.cli.commands
+
+import caseapp.*
+import caseapp.core.help.Help
+
+import scala.cli.commands.BloopExitOptions.parser
+
+// format: off
+final case class SharedDebugOptions(
+  @Group("Debug")
+  @HelpMessage("Turn debugging on")
+    debug: Boolean = false,
+  @Group("Debug")
+  @HelpMessage("Debug port (5005 by default)")
+    debugPort: Option[String] = None,
+  @Group("Debug")
+  @HelpMessage("Debug mode (attach by default)")
+  @ValueDescription("attach|listen")
```

Comment:
It should be added here, that there is available another shorter value, such like this:
```
att|a|lis|l
```
but IMO, we can drop alias `att` for attach mode and `lis`.